src/main/java/net/revelc/code/formatter/xml/lib/FormattingPreferences.java

```diff
@@ -26,6 +28,16 @@
     private int tabWidth = 4;
     private boolean splitMultiAttrs = false;
     private String wellFormedValidation = WARN;
+    private boolean deleteBlankLines = false;
+    
+    public boolean getDeleteBlankLines() {
+	return deleteBlankLines;
+    }
+
+    public void setDeleteBlankLines(Boolean deleteBlankLines) {
+   	if (deleteBlankLines != null)
+   	    this.deleteBlankLines = deleteBlankLines;
+    }
```

Comment:
I think this can just use primitive types. I don't think it needs to be the boxed type. Also, avoid using tab characters to indent code, so that way it uses the same code style as the rest of the project.

```suggestion
    public void setDeleteBlankLines(boolean deleteBlankLines) {
      this.deleteBlankLines = deleteBlankLines;
    }
```